petpals/templates/post.html

```diff
@@ -15,45 +15,34 @@ <h1>Post Discussion</h1>
 
     <div class="container bg-white border border-info border-6 rounded p-4">
       <!--Back Button-->
-      <a href="#" class="btn btn-light btn-sm m-3 has-icon"
-        ><i class="fa fa-arrow-left mr-2"></i>Back</a
-      >
+      <a href="{{ url_for('forum') }}" class="btn btn-light btn-sm m-3 has-icon"><i
+          class="fa fa-arrow-left m-2"></i>Back to the Forum</a>
       <!--end Back Button-->
 
       <!--Original Post Container-->
-      <div
-        class="forum-main-body p-0 collapse forum-content show bg-white mb-3"
-      >
+      <div class="forum-main-body p-0 collapse forum-content show bg-white mb-3">
         <!--Original Post Body-->
         <div class="forum-card-body border border-6 border-light rounded">
           <div class="media forum-item m-3">
-            <a href="#" class="card-link">
-              <img
-                src="https://bootdey.com/img/Content/avatar/avatar1.png"
-                class="rounded-circle"
-                width="50"
-              />
-              <small class="d-block text-center text-muted">Newbie</small>
+            <a href="{{ url_for('profile_router.profile_user', username=post.author.username) }}" class="card-link">
+              <img src="{{ post.author.image_path }}" class="rounded-circle" width="50" />
             </a>
             <div class="media-body ml-3">
-              <a href="#" class="text-secondary">Mokrani</a>
-              <small class="text-muted ml-2">1 hour ago</small>
-              <h5 class="mt-1">Realtime fetching data</h5>
+              <a href="{{ url_for('profile_router.profile_user', username=post.author.username) }}"
+                class="text-secondary"> {{ post.author.username }} </a>
+              <small class="text-muted ml-2">{{ post.timestamp }}</small>
+              <h5 class="mt-1">{{ post.title }}</h5>
               <div class="mt-3 font-size-sm">
-                <p>Hellooo :)</p>
-                <p>
-                  I'm newbie with laravel and i want to fetch data from database in realtime for my dashboard anaytics and i found a solution with ajax but it dosen't work if any one have a simple solution it will be helpful
-                </p>
-                <p>Thank</p>
+                {{ post.content }}
```

Comment:
Doesn't support multiline content.
I recommend checking out the bio on the user/pet template.